### PR TITLE
Allow use of PHP8

### PR DESCRIPTION
**Pull request by dpi**

### CODE REVIEW

**Comment by dpi** [comment]
Packages install fine with this on PHP 8.

No lockfile makes things easier

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": ">=7.2",
```
Comment (kimpepper) [inline composer.json:12]: What about restricting it to `"php": "^7.2||^8.0"` ?

Comment (dpi) [reply composer.json:12]: That works.

There was a thread on twitter, which I cant find now. Where PHP core devs recommended using greater than so they can mass test libs easier. If there truely isnt a rationale for restraining PHP version.

Comment (dpi) [reply composer.json:12]: .. it makes sense for this project :)

**Comment by kimpepper** [comment]
Outdated

**Pull request closed without merging**